src/decisionengine/framework/engine/DecisionEngine.py

```diff
@@ -85,7 +87,7 @@ def block_until(self, state):
     def block_while(self, state):
         with self.workers.unguarded_access() as workers:
             if not workers:
-                self.logger.info('No active channels to wait on.')
```

Comment:
Any reason we're dropping `to wait on` here?

Reply:
I think the 'to wait on' must have been a somewhat new addition? I did not take these out on purpose, so maybe the rebase took them out. Also added back in.